katalog/velero/velero-on-prem/README.md

```diff
@@ -1,11 +1,11 @@
 # Velero on Premises
 
 This Velero deployment is ready to be deployed in any Kubernetes cluster as it includes a MinIO instance compatible with
-the [AWS Velero plugin](https://github.com/vmware-tanzu/velero-plugin-for-aws/tree/v1.1.0).
+the [AWS Velero plugin](https://github.com/vmware-tanzu/velero-plugin-for-aws/tree/v1.5.1).
 
 ## Image repository and tag
 
-- Velero AWS Plugin image: `velero/velero-plugin-for-aws:v1.2.0`
+- Velero AWS Plugin image: `velero/velero-plugin-for-aws:v1.5.1`
```

Comment:
I don't think we use the AWS Plugin for on-prem, do we?

Reply:
We do because for on-prem we rely on MinIO to expose an S3 compliant interface for Velero to use.